testutils/api/client.py

```diff
@@ -27,6 +28,11 @@
 GATEWAY_HOSTNAME = os.environ.get("GATEWAY_HOSTNAME") or "mender-api-gateway"
 
 
+def get_free_tcp_port() -> int:
```

Comment:
Shouldn't this be located somewhere in `testutils` (for example, `testutils/util/network.py`) so backend tests also can access it?